src/main.cc

```diff
@@ -286,11 +292,12 @@ NAN_METHOD(EnumKeys) {
     RegCloseKey(hCurrentKey);
 }
 
-void Init(v8::Handle<v8::Object> exports) {
```

Comment:
Are we able to make the PR diff focused on just this change? The other changes to `nan` abstractions are worth opening in another PR, but the coding style changes are less urgent.

Reply:
Opened #67 for other changes.